packages/apputils/src/toolbar/widget.tsx

```diff
@@ -108,6 +108,8 @@ export namespace Toolbar {
   /**
    * Create a kernel status indicator item.
    *
+   * @deprecated since v3.5
+   * This is dead code now.
```

Comment:
Maybe an alternative comment could be to mention the execution status indicator instead?

Since extension authors might still be using this method in their extension.

Reply:
Also should this be deprecated in 3.6 and removed in 4.0? Or just deprecated?

Reply:
I'm not sure about the plan of removing deprecated functions in this namespace, we have other deprecated items  since 3.2 

https://github.com/jupyterlab/jupyterlab/blob/1d36ce596c48401320e6c33b2a4d17d724db1943/packages/apputils/src/toolbar/widget.tsx#L40-L45

https://github.com/jupyterlab/jupyterlab/blob/1d36ce596c48401320e6c33b2a4d17d724db1943/packages/apputils/src/toolbar/widget.tsx#L61-L66

Reply:
Ah ok, maybe we can do a pass on the docstrings at some point so they are more informative than "dead code" :+1: